src/libs/util.ts

```diff
@@ -0,0 +1,12 @@
+import { clsx, type ClassValue } from 'clsx'
+import { twMerge } from 'tailwind-merge'
+
+/**
+ * @returns merge tailwind is a godsend of a utility
+ * this place is throwing stuff inside
```

Comment:
We should make this function header comment more descriptive for those who are not familiar with Tailwind CSS. Something like "@returns merged Tailwind CSS classes and removes any duplicate classes while avoiding styling conflicts".

Reply:
It's something i will bring up, its not required to use in every of our styles. Just a util but I will add it